pkg/audit/manager.go

```diff
@@ -1013,6 +1013,9 @@ func logViolation(l logr.Logger,
 	constraint *unstructured.Unstructured,
```

Comment:
would we want this to be part of `logConstraint` too?

Reply:
hm, we could but I don't think `logConstraint` is straightforward., at least in its current form, since we don't have the `constraint` struct in hand at the call site anymore:

https://github.com/open-policy-agent/gatekeeper/blob/0fdd27e0790a227e3af2125e7e244fbe3aff4b08/pkg/audit/manager.go#L234-L238

what additional value would we get from adding the constraint annotations here?